tidy3d/plugins/invdes/design.py

```diff
@@ -100,6 +99,21 @@ def initial_simulation(self) -> td.Simulation:
         initial_params = self.design_region.initial_parameters
         return self.to_simulation(initial_params)
 
+    def run(self, simulation, **kwargs) -> td.SimulationData:
+        """Run a single tidy3d simulation."""
+        from tidy3d.web import run
+
+        kwargs.update({"verbose": self.verbose})
```

Comment:
Why `update` instead of `setdefault`? Is the idea to have the class field _always_ take precedence over the passed kwargs?

Reply:
Ah yeah actually I wanted to bring that up but forgot. But now that I think about this again, I think `setdefault` is better. I got confused that if some of our `run` functions set a default value for `verbose`, it will override the class field, which sounded backwards to me. But I realize that the `kwargs` come from the class `to_simulation_data` method, in which case it does make sense that they would take precedence.

Reply:
yeah makes sense. generally my thinking was that explicit parameters (function arguments) should take precedence over implicit ones. as a user I would intuitively expect that